build.sh

```diff
@@ -7,7 +7,7 @@ mkdir ./btrack/obj
 # clone Eigen
 if [ ! -e ./btrack/include/eigen/signature_of_eigen3_matrix_library ]
 then
-  git clone https://gitlab.com/libeigen/eigen.git ./btrack/include/eigen
+  git clone --depth 1 --branch 3.3.9 https://gitlab.com/libeigen/eigen.git ./btrack/include/eigen
```

Comment:
This seems like a good compromise rather than using the old repository. I've just realised that `--branch` here also accepts tags https://stackoverflow.com/a/39067940/7359333. My concern at first was that they may delete the branch.